packages/stark-ui/src/modules/collapsible/components/collapsible-options.intf.ts

```diff
@@ -0,0 +1,29 @@
+/** Possible values for icon position in the collapsible header. */
+export declare type StarkCollapsibleIconPosition = 'left' | 'right' | 'none';
+
+/**
+ * Options related to the Stark Collapsible component
+ */
+export interface StarkCollapsibleOptions {
+
+	/**
+	 * CSS value of the header's height when collapsed.
+	 */
+	collapsedHeight?: string;
+
+	/**
+	 * CSS value of the header's height when expanded.
+	 */
+	expandedHeight?: string;
+
+	/**
+	 * A boolean value triggering the spin of the icon when the collapsible changes status.
+	 */
+	spin?: boolean;
```

Comment:
I think we can keep this feature, it is really handy... but I think we should put it directly as another input in the `StarkCollapsible` component rather than in an options object and rename it to "iconSpinEnabled" or something similar. I know this is less flexible but as I said, the plan is to deprecate this component soon so normally we won't add any other new feature to this component